Makefile.cache

```diff
@@ -69,7 +69,7 @@
 SONIC_COMMON_FILES_LIST :=  $(if $(wildcard cache.skip.common),, .platform slave.mk rules/functions Makefile.cache)
 SONIC_COMMON_FLAGS_LIST :=  $(CONFIGURED_PLATFORM) \
                             $(SONIC_DPKG_CACHE_SOURCE) $(SONIC_DEBUGGING_ON) \
-                            $(SONIC_PROFILING_ON) $(SONIC_SANITIZER_ON)
+                            $(SONIC_PROFILING_ON) $(SONIC_SANITIZER_ON) $(ENABLE_SYNCD_RPC)
```

Comment:
>ENABLE_SYNCD_RPC [](start = 74, length = 16)

which one should use? SONIC_ENABLE_SYNCD_RPC?

Reply:
Fixed. Should follow the same way, using the variable starting with SONIC_